.github/workflows/e2e-manual-multiversion.yml

```diff
@@ -0,0 +1,38 @@
+# Manually run the nightly E2E tests for a particular branch, but test with
+# multiple versions.
+name: e2e-manual-multiversion
+on:
+  workflow_dispatch:
+
+jobs:
+  e2e-manual-multiversion-test:
+    # Run parallel jobs for the listed testnet groups (must match the
+    # ./build/generator -g flag)
+    strategy:
+      fail-fast: false
+      matrix:
+        group: ['00', '01', '02', '03', '04']
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
```

Comment:
That much?

Reply:
This is the same as in https://github.com/tendermint/tendermint/blob/main/.github/workflows/e2e-manual.yml (from which I derived this new workflow). It's also the same as in the nightlies: https://github.com/tendermint/tendermint/blob/main/.github/workflows/e2e-nightly-main.yml#L21

If we should change it, we should probably change it uniformly across all those workflows.

Reply:
This is minor, if that's the value al over, I'd say let's leave it this way